pkgs/development/python-modules/pythondialog/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "pythondialog";
+  version = "3.4.0";
+  disabled = !isPy3k;
```

Comment:
Can't you just combine the `pythondialog` of Python 2 and 3 in one `buildPythonPackage`? Worst case (if they are too different) just in the same file?